pkg/util/datarate/data_rate_test.go

```diff
@@ -36,27 +36,27 @@ func TestDataRate(t *testing.T) {
 		a.So(err, should.BeNil)
 		a.So(string(enc), should.Equal, s)
 
-		var dec udp.DataRate
+		var dec datarate.DataRate
 		err = dec.UnmarshalJSON(enc)
 		a.So(err, should.BeNil)
 		a.So(dec, should.Resemble, dr)
 	}
 
-	var dr udp.DataRate
+	var dr datarate.DataRate
 	err := dr.UnmarshalJSON([]byte{})
 	a.So(err, should.NotBeNil)
 }
 
 func TestValidLoRaDataRateParsing(t *testing.T) {
 	a := assertions.New(t)
 
-	table := map[string]udp.DataRate{
+	table := map[string]datarate.DataRate{
 		"SF6BW125":   {DataRate: ttnpb.DataRate{Modulation: &ttnpb.DataRate_LoRa{LoRa: &ttnpb.LoRaDataRate{SpreadingFactor: 6, Bandwidth: 125000}}}},
 		"SF9BW500":   {DataRate: ttnpb.DataRate{Modulation: &ttnpb.DataRate_LoRa{LoRa: &ttnpb.LoRaDataRate{SpreadingFactor: 9, Bandwidth: 500000}}}},
 		"SF5BW31.25": {DataRate: ttnpb.DataRate{Modulation: &ttnpb.DataRate_LoRa{LoRa: &ttnpb.LoRaDataRate{SpreadingFactor: 5, Bandwidth: 31250}}}},
 	}
 	for dr, expected := range table {
-		actual, err := udp.ParseLoRaDataRate(dr)
+		actual, err := datarate.ParseLoRaDataRate(dr)
```

Comment:
since the package name is `datarate` now you can drop `DataRate` suffix and just make this `datarate.ParseLoRa`